client/hwlib/debian/generate_checksums.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+
+import os
+import hashlib
+import json
+
+def calculate_sha256sum(file_path):
+    """Calculate the SHA-256 checksum of a file."""
+    sha256 = hashlib.sha256()
+    with open(file_path, 'rb') as f:
+        while chunk := f.read(8192):
+            sha256.update(chunk)
+    return sha256.hexdigest()
+
+def generate_checksums(root_dir):
+    """Generate a dictionary of file paths and their SHA-256 checksums."""
+    checksums = {}
+    for dirpath, dirnames, filenames in os.walk(root_dir):
+        # Exclude debian/ and vendor/ directories
+        if 'debian' in dirnames:
```

Comment:
This is okay for now but should only match in the specific two locations. As coded it matches any directory with those names.

Reply:
It should match any files in the directory excluding debian/ and vendor/